### DISCUSSION TITLE
SKR TFT35 E3 V3.0.1 touch screen not updating with SKR E3 V3.0 Motherboard

### DISCUSSION THREAD

**Discussion by Nathaniel125** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2891] [Q&A]
Hi Everyone!

I am working with an Ender 3 Pro that has a new SKR E3 V3.0 motherboard  and SKR TFT35 E3 V3.0.1 display. I have updated the Marlin firmware on both boards and it works well except for this one issue. 

The Marlin Mode (non-touch) interface works as expected. However, when using the display in touch screen mode, the temperature and estep changes I made while updating the mainboard firmware is not reflected in the performance or functionality of the system when using the display in touchscreen mode. For example, when I change the max temp to 305 C, with a 15 C buffer, my touch screen still allows me to raise the nozzle temp to 275, but not 290 as expected (305 - 15 C buffer). The same is true for the bed temp and steps that I also changed.

I learned that the SKR TFT35 E3 V3.0.1 display has its own computer. Therefore I uploaded new firmware and found that the touch screen temp values change when I adjust them in the display firmware settings. However,  it would be more helpful if it reflected the mainboard settings. For instance, I cannot use the extrude functionality on the touchscreen (which I can't figure out how to add to Marlin mode) since the esteps for the extruder isn't adjusted. 

I used the "Marlin SKR 3 mini V3 Bugfix 2.1.x" downloaded 10/23 and "BIGTREETECH-TouchScreenFirmware-master" downloaded 1/24 as parent resources and "BIGTREE_TFT35_V3.0_E3.27.x.bin" for the display firmware. Any insight would be greatly appreciated!

**Comment by radek8** [comment]
The maximum temperature you can set via the display is defined in the config.ini file
The default value is 275°C for the hotend
Try changing the value and updating config.ini
max_temp:T0:275 T1:275 T2:275 T3:275 T4:275 T5:275 BED:150 CHAMBER:60
